gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/crud_service/access_point/AccessPointCrudServiceImpl.java

```diff
@@ -86,7 +86,7 @@ public void updateAccessPoints(
                 var shouldCreateNewAP = true;
                 for (var existingAccessPoint : existingAPsForTarget) {
                     if (
-                        existingAccessPoint.getHost().equals(newAccessPoint.getHost()) &&
+                        existingAccessPoint.getHost().equalsIgnoreCase(newAccessPoint.getHost()) &&
```

Comment:
I was more thinking about implementing a proper equals method on AccessPoint and using it here instead of manually comparing host, secured, and overriding. WDYT?

Reply:
I thought about it. But in this case I think we should ignore case when comparing host.
So, if I override `Object.equals`, I wouldn't respect the contract of the interface.
I can add another method to compare equality ignoring the case of the field `host`, the field `updatedAt` and the field `id`. WDYT?

Reply:
I understand, that to avoid any issue on the repository, I suppose it is better to do your way.  Let's keep it like that 👍